src/components/chip-set/examples/chip-set-filter-badge.tsx

```diff
@@ -0,0 +1,101 @@
+import { Chip } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+const CHIP_SELECET_ALL_ID = 1;
+
+/**
+ * Filter chip set with badge.
+ * The badge can be used to visulize the number of results using each filter.
+ */
+@Component({
+    tag: 'limel-example-chip-set-filter-badge',
+    shadow: true,
+})
+export class ChipSetFilterBadgeExample {
+    @State()
+    private chips: Chip[] = [
+        {
+            id: 1,
+            text: 'All',
+        },
+        {
+            id: 2,
+            text: 'Lime',
+            badgeValue: 5,
+        },
+        {
+            id: 3,
+            text: 'Apple',
+            selected: true,
+            badgeValue: 0,
+        },
+        {
+            id: 4,
+            text: 'Banana',
+            badgeValue: 1,
+        },
+    ];
+
+    @State()
+    private disabled: boolean = false;
+
+    constructor() {
+        this.chipSetOnChange = this.chipSetOnChange.bind(this);
+        this.setDisabled = this.setDisabled.bind(this);
+        this.setAllBadgeValue();
+    }
+
+    public render() {
+        return [
+            <limel-chip-set
+                label="Include fruits of type:"
+                disabled={this.disabled}
+                type="filter"
+                onChange={this.chipSetOnChange}
+                value={this.chips}
+            />,
+            <p>
+                <limel-flex-container justify="end">
+                    <limel-checkbox
+                        label="Disabled"
+                        onChange={this.setDisabled}
+                        checked={this.disabled}
+                    />
+                </limel-flex-container>
+            </p>,
+        ];
+    }
+
+    private chipSetOnChange(event: CustomEvent<Chip>) {
+        console.log(event.detail);
```

Comment:
Is this supposed to be here? :slightly_smiling_face:

Reply:
This is just copied from the example with filter chip-set without the badge. I guess it was there to be able to test the method. 🤔

Reply:
I will remove it, since we do something in the method anyway. 😛